apps/docs/src/app/core/component-docs/table/table-docs.component.html

```diff
@@ -71,7 +71,16 @@
 
 <fd-docs-section-title id="focusable" componentName="table"> Focusable </fd-docs-section-title>
 <description>
-    Table rows and cell can be focusable, by adding <code>[focusable]="true"</code> and <code>tabindex="0"</code> to it.
+    <p>
+        Table rows and cell can be focusable, by adding <code>[focusable]="true"</code> and <code>tabindex="0"</code> to
+        it.
+    </p>
+    <p>Navigation inside table works with arrow keys.</p>
+    <p>Navigating between focusable element inside table cell works with the Tab key.</p>
+    <p>
+        When pressing Tab key inside cell and there's no focusable elements, focus will be switched to the next
+        focusable element outside the table.
```

Comment:
Please mention that if both cells & rows are focusable then only the rows will get focus.